staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2547,6 +2547,8 @@ type ResourceFieldSelector struct {
 // +structType=atomic
 type ConfigMapKeySelector struct {
 	// The ConfigMap to select from.
+	// required
```

Comment:
This is declaring the reference to be required, not the key, but in `validateConfigMapKeySelector` you say it is the key that is covered by declarative.  Is that a mistake or did I miss something?

Reply:
Also, this is `ConfigMapKeySelector` not `ConfigMap` -- It would be the first time we do something underneath Pod, which is a little worrisome.  Did you mean to change ConfigMap itself?